### PR TITLE
made temp vis more dynamic (adjustment for UA07)

### PR DESCRIPTION
**Pull request by KennethLieu**
adjusted the interpolation for temp visualization to account for UA07

### CODE REVIEW

File: clear-lake-restoration-app/src/Components/pages/lakeTChain/LakeTchain.js [hunk 1]
```diff
@@ -331,66 +331,45 @@ export default function LakeTchain(props) {
...
+                    let j = starting+1;
+                    let strings = "Height_" + j + "m";
+                    
+                    //checks the +1 incrementts of the start heights for existance
+                    while (true) {
+                        if (element[strings] == null) {
+                            h = Math.round(j);
+                            break;
+                        } else {
+                            m.push([new Date(element.DateTime_UTC).getTime(),j,parseFloat(element[string])]);
```
Comment (chenalison) [inline clear-lake-restoration-app/src/Components/pages/lakeTChain/LakeTchain.js:359]: element[strings]

**Comment by vtlai88** [comment]
yeah, just pulled your branch and temperature is not working correctly @KennethLieu 
Screen Shot 2022-05-24 at 12 35 24 PM

**Comment by chenalison** [comment]
looks like it works

**Comment by vtlai88** [comment]
yeah, looks good on my end. @CesarAyuso can you check one last time before merging?

**Comment by KennethLieu** [comment]
Resolved conflict with main

**Pull request merged**